create-leo-app/template-offline-public-transaction-ts/.gitignore

```diff
@@ -0,0 +1,7 @@
+
```

Comment:
these .gitignores should not exist this repo is supposed to for generating templates this will likely break the existing mechanism.